bemani/backend/museca/museca1plus.py

```diff
@@ -152,15 +152,26 @@ def enable_event(eid: int) -> None:
 
         if not game_config.get_bool('disable_matching'):
             enable_event(143)  # Matching enabled
-
-        enable_event(1)  # Extended pedal options
-        enable_event(83)  # Light start
-        enable_event(130)  # Curator rank
-        enable_event(195)  # Fictional curator
-        # Event 194 is continuation mode, but it doesn't seem to work on latest data.
-
-        enable_event(98)  # Mission mode
-        for evtid in [145, 146, 147, 148, 149]:
+        event_ids = [
```

Comment:
Oh wow look at all this!

Reply:
MUSECA Plus stuff. I asked Cammy what I need to do and he showed me this

Reply:
Is this safe to use on a vanilla Museca? Do the feature flags do anything there or is it pointless? I ask because this change itself seems a bit unrelated to the rest and could be broken out to its own pull request and merged faster. I have another museca player in the house so it would be fun to get some new things :3